### ISSUE TITLE
[Feature Request] Ask where to save before starting the download

### ISSUE THREAD

**Issue by joaopauloalbq** [davidmhewitt/torrential#151]
For organization reasons, I always download the files in the folders where they should be. 
For example: ~/Videos/Series/Falcon/1 season/

How about an option to ask where to save before starting the download?

**Comment by o-alquimista** [comment]
It looks like a duplicate of https://github.com/davidmhewitt/torrential/issues/64
